### ISSUE TITLE
Implement population local maxima

### ISSUE THREAD

**Issue by jipclaassens** [ObjectVision/2BURP#34]
Temporal constraints of the population (mail of Marcello 22-march):

- If a cell has the highest local population in T1, it can only increase by 3.5K/5K people in T2 (or by [100K - local pop] if this is smaller than 3.5K/5K people) [99 percentile: 3,101 people; 99.5 percentile: 4,822; max: 231,458]
- If a cell is populated but doesn’t have the highest local population in T1, it can only increase in T2 to the local maximum in T1 + 2.5K/3.5K people (or by [100K - local pop] if this is smaller than the local maximum in T1 + 2.5K/3.5 people) [99 percentile: 2,033; 99.5 percentile: 3,435; max: 104,757]
- If a cell is not populated in T1, it can only increase by 350/550 people in T2 [99 percentile: 321; 99.5 percentile: 523; max: 39,482]

**Comment by jipclaassens** [comment]
Furthermore, local (absolute) maxima for pop per unit of are built up based on previous pop and [regional average (convolution) of pop^p] ^(1/p) with p := 2 or 3 and convolution -> 10 km. and/or weight pop by pop^q, ie
[conv (pop^(p+q)) / conv(pop^q)]^(1/p).

**Comment by jipclaassens** [comment]
To determine the maximum allowable population per cell, we applied the local and temporal constraints as suggested by Marcello on March 22nd. However, we encountered a problem. This analysis sets the maximum allowable population density at 100,000 persons per km² of land. Yet, the other three rules specify only an absolute increase (of +3500, +2500, +350 persons). What happens if a cell is only partially built-up, and or less than 1 km2 (as it is at high latitudes)? In our model, we consider densities (people per km² of built-up area that is not non-residential), and if a cell is highgly populated but only partially built-up, the resultant density can be quite high, which can be realistic. However, we detect some ambiguity in the assumed unit of the provided numbers.

Definitions of densities:
1.	Number of people per cell
2.	Number of people per km2 of land
3.	Number of people per km2 of total built-up area
4.	Number of people per km2 of not non-residential built-up area

Are the temporal constraints expressed as densities (and then which definition of density) or number of people per cell? We have now assumed that they express absolute values ().

Proposed solutions:

1.	Reassess the local maxima using density metrics rather than numbers per cell. Note that we have seen unrealistic high densities as discussed in earlier e-mails this week. Which means that the original method probably won’t work on densities. 
2.	Interpret the provided increase numbers (e.g. +3500) as a density value (i.e., an increase of 3500 people per km² of built-up area that is not non-residential per decade).
3.	Convert the provided increase numbers into densities according to definition 4 by using the ratio between the number of highly inhabited cells per region and the amount of not non-residential built-up area in those cells (similar to dasymetric mapping).
